nncf/quantization/quantizer_propagation.py

```diff
@@ -214,8 +232,10 @@ def __init__(self, ip_graph: InsertionPointGraph, ignored_scopes=None):
                     self.QUANTIZATION_TRAIT_NODE_ATTR] = QuantizationTrait.NON_QUANTIZABLE
                 qpg_node[self.AFFECTING_PROPAGATING_QUANTIZERS_ATTR] = []
                 qpg_node[self.OPERATOR_METATYPE_NODE_ATTR] = node[InsertionPointGraph.OPERATOR_METATYPE_NODE_ATTR]
-                scope_node = str(node[InsertionPointGraph.REGULAR_NODE_REF_NODE_ATTR][
-                    NNCFGraph.OP_EXEC_CONTEXT_NODE_ATTR].input_agnostic)
+                iap_context = node[InsertionPointGraph.REGULAR_NODE_REF_NODE_ATTR][
+                    NNCFGraph.OP_EXEC_CONTEXT_NODE_ATTR].input_agnostic
+                qpg_node[self.OPERATOR_SCOPE] = iap_context.scope_in_model
+                scope_node = str(iap_context)
```

Comment:
`node_scope`

Reply:
any other comments to make it worth re-running pre-commit validation?
I can fix it in my next PR's, it looks like a very minor change

Reply:
Not yet, but there will be once I compare this PR and #168; let's wait until then

Reply:
@vshampor any update since then?